docs/cluster_spec.md

```diff
@@ -34,6 +34,34 @@ spec:
       idleTimeoutSeconds: 300
 ```
 
+### etcd v3 & tls
+
+Although kops doesn't presently default to etcdv3 both v3 and TLS can be enabled via the cluster spec (manifests only i.e. no command line options as yet). An upfront warning; at present no upgrade path exists for migrating from v2 to v3 so **DO NOT** try to enable this on a v2 running cluster. The below example snippet assumes a HA cluster of three masters.
+
```

Comment:
Understood that you'd can't upgrade from v2 to v3- is this setting similar to the encrypted etcd options where you MUST enable this at cluster creation? It sounds like it- your doc is wonderfully clear, but I'd add that to your note :)

Reply:
i think encrypted is more absolute though ... Technically with v2 and tls you can enable and then run the numerous commands to migrate and change the peer urls. But i'll chunk it in none the less :-)